pkg/server/options/flags.go

```diff
@@ -80,6 +80,9 @@ var (
 		// KCP Authorization flags
 		"authorization-always-allow-paths", // A list of HTTP paths to skip during authorization, i.e. these are authorized without contacting the 'core' kubernetes server.
 
+		// KCP Admin Authentication flags
+		"authentication-admin-token-store", // Path to which the administrative token hash should be written at startup. If this is relative, it is relative to --root-directory.
```

Comment:
nit: `authentication-admin-token-path` would be little more in-line with other flags

Reply:
Fixed in commit f10ab04